src/main/java/com/gluonhq/richtext/action/DecorateFontWeightAction.java

```diff
@@ -0,0 +1,47 @@
+package com.gluonhq.richtext.action;
+
+import com.gluonhq.richtext.RichTextArea;
+import com.gluonhq.richtext.model.TextDecoration;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.value.ChangeListener;
+
+import static javafx.scene.text.FontWeight.BOLD;
+import static javafx.scene.text.FontWeight.NORMAL;
+
+public class DecorateFontWeightAction extends DecorateAction<Boolean> {
```

Comment:
That's true. But, in that case we will have a `ComboBox<T>` where `T` is `FontWeight`. In case of a ToggleButton, its not possible to bind it to a particular type.